lib/matplotlib/axes/_axes.py

```diff
@@ -5519,6 +5519,9 @@ def imshow(self, X, cmap=None, norm=None, aspect=None,
             Note that the vertical axes points upward for 'lower'
             but downward for 'upper'.
 
+            See the example :doc:`/tutorials/intermediate/imshow_extent` for
+            examples and a more detailed description.
```

Comment:
```suggestion
            a more detailed description.
```

I would keep this exactly the same as in the sentence in the `extent`parameter description. Twice using example is kind of redundant. :smile: 

https://21987-1385122-gh.circle-artifacts.com/0/home/circleci/project/doc/build/html/api/_as_gen/matplotlib.axes.Axes.imshow.html#matplotlib.axes.Axes.imshow

Reply:
I'll keep them consistent but I think the current spelling is actually a little confusing.